introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello, my name is David Snyder, and I work at Ventiv Technologies (www.ventivtech.com).
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?